src/sql/workbench/contrib/query/browser/queryInputFactory.ts

```diff
@@ -26,7 +26,7 @@ const editorInputFactoryRegistry = Registry.as<IEditorInputFactoryRegistry>(Edit
 
 export class QueryEditorLanguageAssociation implements ILanguageAssociation {
 	static readonly isDefault = true;
-	static readonly languages = ['sql'];
+	static readonly languages = ['sql', 'kusto'];
```

Comment:
It is to enable .kql extension files to be treated as Query files in ADS so that actions toolbar on the top shows up and functions. This change is not intended for master branch but for local Kusto branch which I am working on.

Reply:
If you're not intending on bringing this into master then yeah you can do whatever, although I'm curious what the intention of this work is then if you're not planning on merging back into master at some point. 

And if you do bring it into master at some point I'm thinking some more work will be needed to be done to make this work correctly - I'm assuming most of the functionality here in the query editor isn't going to work for KQL. So something to think about if you do indeed on actually publishing this extension.

Reply:
This will eventually come to master, once we have basic functionality in place. Monica, please work with Charles regarding having this hooked up in the right place.